webservices/tasks/legal_docs.py

```diff
@@ -38,6 +38,9 @@
     ORDER BY case_serial
 """
 
+# SLACK_BOTS = "#bots"
+SLACK_BOTS = "#test-bot"
```

Comment:
Should this be `"#bots"` before merge?

Reply:
Yes, will revert to #bots once testing is done.